amps/share-services/pom.xml

```diff
@@ -32,10 +32,6 @@
                     <groupId>javax.servlet</groupId>
                     <artifactId>jstl</artifactId>
                 </exclusion>
-                <exclusion>
-                    <groupId>commons-collections</groupId>
-                    <artifactId>commons-collections</artifactId>
```

Comment:
Removed exclusion due to failing test:
`Error:    ReadOnlyTransactionInGetSlingshotApiTest.testGetDoclibCategoryNode:208->BaseWebScriptTest.sendRequest:321->BaseWebScriptTest.sendRequest:372 Status code 500 returned, but expected 200 for /slingshot/doclib/categorynode/node/workspace/SpacesStore/61b4570b-f000-42d5-b17a-e8528198673d (get)`
From stack trace:
`Caused by: java.lang.ClassNotFoundException: org.apache.commons.collections.CollectionUtils`

Reply:
I think these exclusions may have been important as we seem to be setting the version of `commons-collections` in the main POM' `dependencyManagement` explicitly (and we set it to the newest version which is vulnerability free). 
On the other hand - it doesn't look that your changes have introduced this error.
I'd say that we need to take care of that in Webscripts project but maybe exclusion on that Webscript dependency with `test` scope would be enough?

Reply:
Dependency scope changed to: `commons-collections:commons-collections:jar:3.2.2:test`